layout/_macro/post.swig

```diff
@@ -56,28 +56,16 @@
                     <i class="fa fa-thumb-tack"></i>
                   </span>
                 {% endif %}
-                <a class="post-title-link" href="{{ url_for(post.path) }}" itemprop="url">
-                  {{ post.title | default(__('post.untitled'))}}
-                </a>
-              {% else -%}
-                {{- post.title -}}
-                {% if theme.post_edit.enable -%}
-                  <a href="{{ theme.post_edit.url }}{{ post.source.slice(7) }}" class="post-edit-link" title="{{ __('post.edit') }}" target="_blank">
```

Comment:
It seems u breaking `post_edit.url` option which was added in #208 pull and some additional commits.

What u can do ith this cases?

1. Stash your current changes.
2. Pull master branch to your master (or any other worked branch) with latest changes.
3. Pop your current changes and try to resolve all possible conflicts (if they will appear).

Reply:
Oops, i didn't notice it.
My git is already pull from master's latest commit height,
may i just overwrite my two git into one?

Reply:
> Oops

Yeah, i know this feeling too.. Awaiting for fixed changes.